### ISSUE TITLE
Github Links in Slack messages are broken

### ISSUE THREAD

**Issue by joelwatson** [TeamCodeStream/CodeStream#6]
When receiving Github bot messages in Slack, the urls in CodeStream are somewhat broken. For example:

```
<https://github.com/somerepo/compare/{uuid}...{uuid}|1 new commit> pushed to https://github.com/somerepo/tree/master|`master`
https://github.com/somerepo/commit/{uuid}|`{commit##}` - Update build.sh
```

So in the above, the first part is linked, but includes the "|1": 
```https://github.com/somerepo/compare/{uuid}...{uuid}%7C1```

The other links suffer from similar issues, including `build.sh` simply linking to `build.sh`

CS Extension Version: 0.15.0
VS Code Version: 1.28.0

**Comment by joelwatson** [comment]
I imagine this is related, but I've noticed that ClubHouse ticket links exhibit the same issue:

```[Workspace] SomeUser commented on <https://app.clubhouse.io/myclubhouse/story/xxxx|My Story with Spaces>:```

In this, the following will be linked:

```https://app.clubhouse.io/myclubhouse/story/xxxx|My```

URL regexes are no fun :(

**Comment by planteater** [comment]
Fixed in 0.17

**Comment by planteater** [comment]
Closing this, as support for stream-based comminucation within CodeStream is no longer a priority.
